tests/5.0/teams/test_teams_distribute_default_none.c

```diff
@@ -0,0 +1,76 @@
+//===--- test_teams_default_none.c------------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
+//
+// This tests uses the default(none) clause on a teams directive.
+// The test aims to validate that all values will not have default data sharing
+// attributes.
+//
+////===----------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 128
+
+int main() {
+  int a[N];
+  int b[N];
+  int c[N];
+  int d[N];
+  int x;
+  int privatized;
+  int num_teams;
+  int share = 0;
+  int errors = 0;
+
+  OMPVV_WARNING("Test only uses default(none) clause and does not guarantee that the default(none) is enforced.");
+
+  for (int x = 0; x < N; ++x) {
+    a[x] = 1;
+    b[x] = x;
+    c[x] = 2*x;
+    d[x] = 0;
+  }
+
+#pragma omp teams distribute num_teams(4) default(none) shared(a, b, c, d, num_teams) private(privatized)
```

Comment:
Wondering if this is compliant...
4.5: "If specified, a teams construct must be contained within a target construct. That target construct must not contain any statements or directives outside of the teams construct."

5.0: "A teams region can only be strictly nested within the implicit parallel region or a target region. If a teams construct is nested within a target construct, that target construct must contain no statements, declarations or directives outside of the teams construct."

Reply:
I don't understand the comment. We are not using the target construct. IF it was nested then the 5.0 part would apply. This is a standalone teams construct.

Reply:
@jhdavis8 @tmh97 review complete ?

Reply:
Ah I see, I think I was mistinterpreting the spec to say standalone teams was not permitted. Will approve.